src/plays/countries-statics/Country.jsx

```diff
@@ -0,0 +1,160 @@
+import React, { useEffect, useState } from "react";
+
+export default function Country({ activeGeo }) {
+  const [country, setCountry] = useState();
+  const [isloading, setIsLoading] = useState(false);
+  const [errorInfo, setErrorInfo] = useState(null);
+  const getCountry = () => {
+    setIsLoading(true);
+    fetch(`https://restcountries.com/v3.1/alpha/${activeGeo}`)
+      .then((response) => {
+        if (!response.ok) {
+          throw Error("Something went wrong");
+        }
+        return response.json();
+      })
+      .then((data) => {
+        setIsLoading(false);
+        setErrorInfo(null);
+        setCountry(data[0]);
+      })
+      .catch((e) => {
+        setIsLoading(false);
+        setCountry([]);
```

Comment:
This should be `setCountry()` as in downstream process you are referring `country && (`. An empty array will pass the condition during rendering